lib/kitchen/verifier/shell.rb

```diff
@@ -36,16 +36,23 @@ class Shell < Kitchen::Verifier::Base
       default_config :command, "true"
       default_config :shellout_opts, {}
       default_config :live_stream, $stdout
+      default_config :remote, false
```

Comment:
Because of that, I used `enter_instance` to option name of sample code.
But it might be good to use `transport` to option name.